chispa/rows_comparer.py

```diff
@@ -1,24 +1,26 @@
+from __future__ import annotations
+
 from itertools import zip_longest
 
 from prettytable import PrettyTable
 
 import chispa
-from chispa.default_formats import DefaultFormats
-from chispa.terminal_str_formatter import format_string
+from chispa.formatting import FormattingConfig, TerminalStringFormatter
 
 
-def assert_basic_rows_equality(rows1, rows2, underline_cells=False, formats=DefaultFormats()):
+def assert_basic_rows_equality(rows1, rows2, underline_cells=False, formats=FormattingConfig()):
```

Comment:
I do not like the idea of using mutable objects as default values for functions without a reason. Why cannot we pass it as an Optional with a default equal to None. And if it is None, we can create it in the body.

Reply:
Good idea, I will implement that!